data/helper/data_dictionary.csv

```diff
@@ -14,6 +14,7 @@ Person,alt_name,Alias limited to one pseudonym per person. Change all traditiona
 Person,alt_name_lang,key: IsoLangCode.iso_code,,2017-07-04,LH,2018-02-01,DP
 Person,place_of_birth,key: Place.place_ID 'PL0999' in case we do not know the place.,,2017-07-04,LH,2018-11-13,MG
 Person,social_position,Social position of a reader; 'unknown' in case you do not know; 'NULL' for authors and other non-readers; (worker / party-cadre /…),,2017-07-04,LH,2018-01-30,DP
+Person,narrative_age,Narrative age of fictional personas only: 01: childhood / 02: mid-age / 03: old-age. Add 'NULL' in case this does not apply (real life persons),,2019-06-18,MG,2019-06-18,MG
```

Comment:
```suggestion
Person,narrative_age,Narrative age of fictional personas only: 01: childhood / 02: mid-age / 03: old-age. 'NULL' or empty in case this does not apply (real life persons) 00 if unknown,,2019-06-18,MG,2019-06-18,MG
```